src/App.js

```diff
@@ -66,29 +66,32 @@ const App = () => {
     <AuthProvider>
       <Router>
         {/* <Header toggleTheme={toggleTheme} theme={theme} /> */}
-        <DocumentBody>
-          <SideHeader />
-          <Main theme={theme}>
-            <Switch>
-              <Route path="/about">
-                <About theme={theme} />
-              </Route>
-              <Route path="/login">
-                <LogIn theme={theme} />
-              </Route>
-              <Route path="/register">
-                <Register theme={theme} />
-              </Route>
-              <Route path="/add">
-                <AddNew data={state.data} theme={theme} />
-              </Route>
-              <Route path="/">
-                <Home data={state.data} theme={theme} />
-              </Route>
-            </Switch>
-          </Main>
-          {/* <Footer theme={theme} /> */}
-        </DocumentBody>
+        <ThemeProvider theme={darkTheme}>
```

Comment:
I think we can set theme with the useState above, and then use state.theme here or such, but yeah lets merge and we can use this provider 👍  !

Reply:
Last week I set it with ThemeProvider and useState. It worked nicely. But it doesn't work anymore, I'll find out why